extended_tck.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
after running this script I have a stopped container and its image left on the box
```
 dev-cloudstate-tck:local 
```
I think we can at least remove the container.